mcserver/settings.py

```diff
@@ -205,3 +206,58 @@
 OTP_EMAIL_BODY_TEMPLATE_PATH = 'email/verification_email.html'
 OTP_EMAIL_SUBJECT = ""
 OTP_EMAIL_BODY_TEMPLATE = ""
+
+
+# Sentry support
+
+SENTRY_DSN = config('SENTRY_DSN', default='')
+
+if SENTRY_DSN:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    from sentry_sdk.integrations.celery import CeleryIntegration
+    from sentry_sdk.integrations.redis import RedisIntegration
+
+
+    def strip_sensitive_data(event, hint):
+        """ This function removes the DisallowedHost errors from
+        the Sentry logs for avoiding excedding the quota.
+        """
+        if 'log_record' in hint:
+            if hint['log_record'].name == 'django.security.DisallowedHost':
+                return None
+        return event
+
+
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        integrations=[
+            DjangoIntegration(),
+            CeleryIntegration(),
+            RedisIntegration()],
+        before_send=strip_sensitive_data,
+        send_default_pii=True
+    )
+
+REDIS_URL = config('REDIS_URL')
```

Comment:
@olehkorkh-planeks looks like I should add this to my .env variable, what should it be?